Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:20.04
```

Comment:
I expected this to be a lot heavier, but the final image size, even with the full application, is only 485MB. I'm open to trying to slim that down if it's still too big.

Reply:
You can reduce the image to 200MB by remove the `apt install` and doing `FROM python:3.8-slim` (might as well update to 3.9 or 3.10), as well as using `pip3 install --no-cache-dir -r /tmp/requirements.txt`

Reply:
couldn't find any 3.10 slim images that had gcc, so going with `3.9-slim`, which "just works" (:tm: )